### ISSUE TITLE
Score page should be aligned to the middle when opened

### ISSUE THREAD

**Issue by jessjwilliamson** [musescore/MuseScore#9697]
**Describe the bug**
I'm not sure if this issue has been reported yet.
Currently, the score is being aligned top left corner when opened/created new score. It should be aligned in the middle as MS3 does.

**Expected behavior**

![image](https://user-images.githubusercontent.com/71656189/141139016-a1b3b4e2-75b1-4632-b611-67a260b93942.png)

**Screenshots**

![image](https://user-images.githubusercontent.com/71656189/141139105-fe989221-bc54-4c1c-906d-7a5dd08a81f0.png)

**Desktop (please complete the following information):**
 - OS: Windows, MacOS

**Comment by MarcSabatella** [comment]
There was some discussion of this in the forum when the change from left to centering was made, not so long ago.  As I recall there was some strong opposition.  To me, it's good to be consistent, to have the page in the same position whether just opening it, using the Home button, scrolling, or having it automatically scroll in playback.  In 3.6.2 currently, these are *not* all the same, resulting in the score jumping around horizontally during use.  Quite distracting for my demonstrations, actually.

**Comment by MarcSabatella** [comment]
See https://musescore.org/en/node/317227 for the priot discussion.  @cbjeukendrup you had prepared a fix of some kind of 3.x branch?

**Comment by cbjeukendrup** [comment]
Yes, Marc is right. I had implemented this for 3.6, but some users complained about it, so I reverted it for the unreleased 3.6.3. 

However, I myself still like the idea. I can probably implement it somewhat more thoroughly for 4.0, by creating a smart behaviour when the size of the score view changes (for example, when just resizing the window, but also when docking a panel to the right side of the window). Normally, the old center of the view should move to the new center, except when that would move the score out of view or create more "blue area". We should make this nicely consistent with the behaviour on zooming.

So, I think we can just implement it for MU4, and then see what users think during alpha/beta phase. Maybe, they have meanwhile got used to it in 3.6.2.

**Comment by MarcSabatella** [comment]
As I see it, the fact that the mixer or other side panels can cover the score or that it is not re-centered is just a bug to be fixed, not a flaw in the idea of centering.  The idea that it moves the score further from the palettes - cadiz1's point - is maybe harder to deal with.

**Comment by cbjeukendrup** [comment]
Yeah, but I really wonder whether that is a serious problem... on most screens, it will be just a few hundreds of pixels extra distance from the palettes. Also, many/most scores are more than one page, so will likely exceed the width of the score view, so they will be just left aligned, as in MS 3.6.2.

**Comment by Tantacrul** [comment]
Agree we should do this. It looks weird and broken to have it zoomed out and moved to the left of the screen.

**Comment by MarcSabatella** [comment]
Not sure what happened to the comments here, but *please* look at the history behind this.  I personally prefer it centered as well, but there are very strong arguments for alternatives and some way should be found to accommodate them.

**Comment by Tantacrul** [comment]
If someone feels strongly that the app should look wrong when a score opens, despite everyone who I know who has sense thinking the opposite, they can continue using 3.6

All 2 of them.

**Comment by Tantacrul** [comment]
I deleted the comments because I'm trying to unconfuse our developers and reduce cognitive load.

**Comment by MarcSabatella** [comment]
This is not what anyone has said.  Please, listen to our users to understand their concerns.  It has to do with use cases where you want the score close to the palette to minimize mouse motion, and there would be ways of designing a solution that allows for that while still allowing the default to center.  It’s OK to decide not to implement such a solution for 4.0. It’s not OK to delete all evidence that such a conversation ever took place.  That is not how open source works.

**Comment by Tantacrul** [comment]
I must say, that is completely ridiculous. Utter and total nonsense, actually. Sorry. I just don't know what else to say.

It's basically a theoretical or outright fake 'concern' raised to be a nuisance.

**Comment by Tantacrul** [comment]
Closing comments and removing the last few to keep the task simple.

**Comment by cbjeukendrup** [comment]
Fixed by https://github.com/musescore/MuseScore/pull/24483
